docs/layers/identity/centralized-terraform-access.mdx

```diff
@@ -36,21 +30,185 @@ assume the role in any account.
+
+```mermaid
+flowchart LR
+    subgraph external_idp["External Identity Provider"]
+        user1["User 1"]
+        user2["User 2"]
+        user3["User 3"]
+        group1["Group 1"]
+        user1 --> group1
+        user2 --> group1
+        user3 --> group1
+    end
+
+    subgraph aws_iam_identity_center["AWS IAM Identity Center"]
+        user1_copy["User 1 Copy"]
+        user2_copy["User 2 Copy"]
+        user3_copy["User 3 Copy"]
+        group1_copy["Group 1 Copy"]
+        permissions["Permission Sets"]
+        user1_copy -.-> group1_copy
+        user2_copy -.-> group1_copy
+        user3_copy -.-> group1_copy
+        group1_copy --> permissions
+    end
+
+    external_idp -->|Automatic Provisioning| aws_iam_identity_center
+```
+
+### Access the AWS Web Console with Identity Center
+
+With Identity Center, it’s very easy to log into the AWS web console. Simply open the Single Sign On Portal link, and select the account and role you want to access.
+
+For example, if I wanted to connect to `plat-prod` with “Read Only” access, then I would select the `ReadOnlyAccess` Permission Set for `plat-prod`. AWS then will launch a new session with the appropriate permissions in production
 
-With AWS IAM Identity Center, you can directly access a specific account from the single sign-on page, which will automatically assume the role with the associated `PermissionSet`. This is great for web console usage but not ideal for locally applying Terraform. With Terraform, we need to apply changes across many accounts, where each account has its own role. We don't want to assume a new role each time we change accounts. To solve this, we have a specific Permission Set that can assume an AWS team role in the identity account. When applying Terraform, we assume that "team" role via SSO, and when accessing the web console, we use the Permission Set directly.
 
-### AWS SAML
+### Deploying both AWS IAM Identity Center and AWS SAML
+
+AWS IAM Identity Center and AWS SAML are not mutually exclusive solutions. They can be used together to provide a seamless login experience for users. Cloud Posse frequently deploys both in customer environments - AWS SAML to log in ourselves for the duration of the engagement and AWS IAM Identity Center for the customer to use and support. That way the customer can access AWS via their chosen Identity Provider (IdP), and we can use GSuite SAML to access the Identity account team directly. When an engagement is complete, we can easily decommission our access by removing the SAML connection.
+
+:::tip AWS IAM Identity Center or AWS SAML? Which do I choose?
+
+The vast majority of our customers prefer AWS IAM Identity Center (SSO). The convenience of a web console login is hard to beat. However, some customers prefer SAML for its simplicity and compatibility with existing systems. We support both methods, and you can choose the one that best fits your needs.
```

Comment:
I think the part we gloss over is that you always deploy `aws-saml` because it's how the identity architecture is implemented for automation. The option is whehter or not they also deploy AWS SSO.

Reply:
we dont need `aws-saml` for automation. GH uses OIDC to assume the team directly - saml isnt involved

Reply:
GH OIDC as a whole isnt explained in this layer. We should document somewhere specifically how it works - assuming a team from github, using a mixin, github-oidc-role, or something else, etc

We have this, but it's about OIDC as a process not for our architecture
https://docs.cloudposse.com/layers/github-actions/github-oidc-with-aws/